### PR TITLE
Updating getCategory api contract

### PR DESCRIPTION
**Pull request by tavisca-rrai**
@tavisca-smohite @achirametavisca

### CODE REVIEW

File: Contracts/categories/v1.0/get-categories/error-response.json5 [hunk 1]
```diff
@@ -1,21 +1,5 @@
...
-{
-	"code":"404",
-	"message":"Not found",
-	"info":[
-		{
-			"code":"404",
-			"message":"Requested data is not found"
-		}
-	]
+	"status code": "400",
```
Comment (Tavisca-smohite) [inline Contracts/categories/v1.0/get-categories/error-response.json5]: Error response has to be consistent with all api responses

File: Contracts/categories/v1.0/get-categories/response.json5 [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
+            "name":"Toys"
+        },
+        {
+            "id":"6482731",
+            "name":"Books"
+        }
+    ],
+
+    "status code" : "1231414"
+
```
Comment (Tavisca-smohite) [inline Contracts/categories/v1.0/get-categories/response.json5]: Why is this needed?

**Review by Tavisca-smohite — CHANGES_REQUESTED** [review]

**Comment by Tavisca-smohite** [comment]
Also categories will be singular or plural?

**Comment by tavisca-rrai** [comment]
> Also categories will be singular or plural?

okay

File: Contracts/categories/v1.0/get-categories/error-response.json5 (hunk 1 above)
Comment (tavisca-rrai) [reply Contracts/categories/v1.0/get-categories/error-response.json5]: done

File: Contracts/categories/v1.0/get-categories/response.json5 (hunk 2 above)
Comment (achirametavisca) [reply Contracts/categories/v1.0/get-categories/response.json5]: @tavisca-rrai 
Is it done ?
Please update.

Comment (tavisca-rrai) [reply Contracts/categories/v1.0/get-categories/response.json5]: yes, it is updated

**Review by achirametavisca — APPROVED** [review]

**Review by tavisca-swapnil — APPROVED** [review]
